package.json

```diff
@@ -13,14 +13,18 @@
     "./package.json": "./package.json",
     "./path-templating.bnf": "./src/path-templating.bnf"
   },
+  "watch": {
+    "test": "{src,test}/*.js"
+  },
   "scripts": {
     "prepublishOnly": "npm run build",
     "grammar:compile": "./node_modules/apg-js/bin/apg.sh --lite --in=./src/path-templating.bnf --out=./src/path-templating.js && cd ./src",
     "build": "npm run grammar:compile && npm run build:es && npm run build:cjs && npm run build:cjs:apg-lite",
     "build:es": "cross-env BABEL_ENV=es babel src --out-dir es --extensions '.js' --out-file-extension '.mjs'",
     "build:cjs": "cross-env BABEL_ENV=cjs babel src --out-dir cjs --extensions '.js' --out-file-extension '.cjs'",
     "build:cjs:apg-lite": "cross-env BABEL_ENV=cjs babel node_modules/apg-lite/lib/parser.js --out-file ./cjs/apg-lite.cjs",
-    "test": "mocha"
+    "test": "mocha",
+    "watch": "npm-watch"
```

Comment:
```suggestion
    "test:watch": "npm-watch test"
```

Reply:
Doing this would result in not being able to watch other scripts like build etc. (Which script to watch can be passed as an argument) 

I don't have strong feelings here, but I thought it might be interesting to watch different things if needed

Reply:
Understood, let's compromise on https://github.com/swaggerexpert/openapi-path-templating/pull/144/files#r1867713023. I want to be able just to run watch on tests individually. Would that work with you?

Reply:
that works! Thank you!